### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by wow-such-amazing**
Sorry for pushing everything in 1 commit 🙃 
Updated project and workspace to swift 4.2;
Auto fixed projects' warnings generated by xcode;
Changed podfile. Now example project will always use code from local folder;
Fixed autolayout warnings in the console for table view cells;
Changed library version in podspecto 2.1.3;

### CODE REVIEW

**Comment by wow-such-amazing** [comment]
Checks failed because it can't be compiled with Xcode 9

**Comment by wow-such-amazing** [comment]
@ayushn21 Can you please merge any request that updates this pod to swift 4.2? 🙂

**Comment by ayushn21** [comment]
@crabman448 Yes! I will do it tonight and publish the update to cocoapods. I really need to sort out my notifications; I don't get emails about PRs and issues so I completely missed this!

**Comment by wow-such-amazing** [comment]
@ayushn21 Thanks! No worries)

File: AvatarImageView.podspec [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name              = "AvatarImageView"
-  s.version           = "2.1.2"
+  s.version           = "2.1.3"
```
Comment (ayushn21) [inline AvatarImageView.podspec]:
```suggestion
  s.version           = "2.2"
```

This should be 2.2 as it contains breaking changes :)

**Comment by ayushn21** [comment]
Pretty much everything looks good. Great stuff. Thanks for making these updates!

Just one more thing. Could you please update the Xcode version in `.travis.yml` to `xcode10` so the tests can run before I merge this?

**Comment by wow-such-amazing** [comment]
hmm 🤔

**Comment by ayushn21** [comment]
These failures are more likely to be a quirk with the way the tests are written. I should really be using the Facebook screenshot testing library. I think I'll risk it and merge and just do some manual testing to make sure nothing's broken.

Thanks 👍

**Comment by wow-such-amazing** [comment]
Cool, thanks :)

**Comment by ayushn21** [comment]
Swift 4.2 is now supported in v2.2 which has also been pushed to CocoaPods :D

**Comment by wow-such-amazing** [comment]
Awesome!

**Pull request merged**